util/helpfulIO.py

```diff
@@ -84,9 +88,13 @@ def setPosition(self, pos: int) -> None:
         if RobotBase.isReal():
             self.motor.set(ControlMode.Position, pos)
         else:
-            positionChange = pos - self.simEncoder
-            change = self.pidController.calculate(positionChange)
-            self._setSimMotor(change)
+            change = self.pidController.calculate(self.simEncoder, pos)
+            print(
```

Comment:
print should not be spammed

Reply:
Sim only, also your code not mine smh

Reply:
sim only is still intensive when you're doing a bunch of other stuff together